packages/devtools_app/test/performance/enhance_tracing_test.dart

```diff
@@ -2,12 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'package:devtools_app/src/config_specific/ide_theme/ide_theme.dart';
+import 'package:devtools_app/devtools_app.dart';
```

Comment:
per a previous [discussion](https://github.com/flutter/devtools/pull/4566#discussion_r999849325) on another PR, it is acceptable to import devtools_app.dart into test files.